runtime/src/main/java/io/quarkus/vault/runtime/config/VaultBootstrapConfig.java

```diff
@@ -228,6 +228,18 @@ public class VaultBootstrapConfig {
     @ConfigItem
     public Optional<List<String>> nonProxyHosts;
 
+    /**
+     * The proxy host. If set the client is configured to use a proxy.
+     */
+    @ConfigItem
+    public Optional<String> proxyHost;
+
+    /**
+     * The port the proxy is listening on, 3128 by default.
+     */
+    @ConfigItem(defaultValue = "3128")
+    public Optional<Integer> proxyPort;
```

Comment:
if there is a default value, why would we make it optional?

Reply:
Good remark ... also integrated in new commit